components/server/src/user/user-service.ts

```diff
@@ -155,7 +155,32 @@ export class UserService {
      * @param date The date for which we want to know the default workspace timeout
      */
     async getDefaultWorkspaceTimeout(user: User, date: Date = new Date()): Promise<WorkspaceTimeoutDuration> {
-        return "30m";
+        return WORKSPACE_TIMEOUT_DEFAULT_SHORT;
+    }
+
+    public workspaceTimeoutToDuration(timeout: WorkspaceTimeoutDuration): string {
+        switch (timeout) {
+            case WORKSPACE_TIMEOUT_DEFAULT_SHORT:
+                return "30m";
+            case WORKSPACE_TIMEOUT_DEFAULT_LONG:
+                return "60m";
+            case WORKSPACE_TIMEOUT_EXTENDED:
+            case WORKSPACE_TIMEOUT_EXTENDED_ALT:
+                return "180m";
+        }
+    }
```

Comment:
Aha, I see that you've made Gitpod Enterprise Edition configurable, but the non-Enterprise edition still has only hard-coded values. Just to be overly safe: That's intended, right?

Reply:
That's intended in that way that it is similar to what we have before. If it makes sense or not is more a business decision (see also [this internal discussion](https://gitpod.slack.com/archives/C01KGM9DVRC/p1646666523461289)).